ctk/common/src/main/kotlin/org/codice/compliance/verification/core/responses/CoreLogoutResponseProtocolVerifier.kt

```diff
@@ -42,10 +42,11 @@ class CoreLogoutResponseProtocolVerifier(logoutRequest: LogoutRequest, samlRespo
                 ?.firstOrNull()
```

Comment:
This is out of scope of this PR, but I just had the thought of adding a `firstChild` extension function. That would simplify this logic significantly, and I wonder if it would simplify a lot of other places in the code as well.